src/fragments/lib/logging/common/sending_logs/sending_logs.mdx

```diff
@@ -0,0 +1,20 @@
+The Amplify library already uses the Amplify Logger internally.  Each Amplify feature logs messages at various log levels.  You can log your own custom messages and various log levels depending on your use cases.  Only log levels that are the same or below the log level defined in the logging constraints are sent to Amazon CloudWatch.
+
+## Log an message
+Get an instance of the `Logger` and specifying a `category name` and/or `namespace`.  Use the `Logger` instance to log messages at the desired log level.  The `category name` and `namespace` values are used to tag your log messages that will appear in CloudWatch.  It is also use to identify and fetch the logger again without creating a new `Logger` instance.
```

Comment:
You can log messages using the Amplify logger to a specific namespace to help you group logs that are similar when they are sent to CloudWatch. To accomplish this, Get an instance of the Logger and specify a category name and/or namespace. Use the Logger instance to log messages at the desired log level. The category name and namespace values are used to tag your log messages that will appear in CloudWatch. It is also use to identify and fetch the logger again without creating a new Logger instance.

Reply:
Description text updated.